### ISSUE TITLE
Upgrade to 3.5a failed

### ISSUE THREAD

**Issue by mikev-junimo** [conda-forge/tmux-feedstock#36]
### Solution to issue cannot be found in the documentation.

- [x] I checked the documentation.

### Issue

Hi,

I see that commit https://github.com/conda-forge/tmux-feedstock/commit/bd4ab6bf3801657a2902986e38f28672e7175439 upgraded this to 3.5a but the CI failed so I don't think the upgrade got pushed.

When I install this package I still get 3.5 instead of 3.5a.
cc @seowalex

### Installed packages

```shell
-
```

### Environment info

```shell
-
```

**Comment by seowalex** [comment]
@matthiasdiener  is right, and you can see how Conda orders version strings [here](https://github.com/conda/conda/blob/main/conda/models/version.py#L52). If you want 3.5a, you can explicitly choose it (which will warn you that you are "downgrading" tmux):

```bash
conda install tmux==3.5a
# or
mamba install tmux==3.5a
```

**Comment by seowalex** [comment]
Something interesting is that Conda suggests this workaround:

> You can work-around this problem by appending an underscore to plain version numbers:
> 1.0.1_ < 1.0.1a =>  True   # ensure correct ordering for openssl

Although I'm not sure of any feedstock that has actually done this, and it wouldn't apply to 3.5 retroactively anyway.

**Comment by mikev-junimo** [comment]
Thanks for the info, this makes sense. I have `3.*` in my env file. I will update it to use the explicit version.

**Comment by matthiasdiener** [comment]
> [@matthiasdiener](https://github.com/matthiasdiener) is right, and you can see how Conda orders version strings [here](https://github.com/conda/conda/blob/main/conda/models/version.py#L52). If you want 3.5a, you can explicitly choose it (which will warn you that you are "downgrading" tmux):
> 
> conda install tmux==3.5a
> # or
> mamba install tmux==3.5a

Great catch @seowalex! Would it make sense to "rename" 3.5a to something like 3.5post_a or so?

**Comment by seowalex** [comment]
That's possible, but it might be confusing for people looking for `3.5a` in particular. What do you think?

**Comment by matthiasdiener** [comment]
> That's possible, but it might be confusing for people looking for `3.5a` in particular. What do you think?

Yeah, you're right. Perhaps we should rather rename `3.6` to `3.6_` when it is released?

**Comment by XhstormR** [comment]
This error is annoying and causes me to not be able to use this command `micromamba update -a`, and every time I run the command, it becomes the old version.

**Comment by XhstormR** [comment]
Version 3.6a has been released. https://github.com/tmux/tmux/releases/tag/3.6a

**Comment by XhstormR** [comment]
Is there a way to automatically detect new version releases by checking tags in chronological order?

https://github.com/tmux/tmux/tags
